packages/client/src/file.ts

```diff
@@ -1,12 +1,14 @@
-import { ContentUri } from "../wrappers/content-uri"
-import { ContentHashedUri } from "../wrappers/content-hashed-uri"
-import { IRequestParameters } from "../net/gateway-dvote"
-import { IGatewayDVoteClient } from "../common"
-import { IPFS } from "../net/ipfs"
+import {
+    ContentHashedUri,
+    ContentUri,
+    IGatewayDVoteClient,
+    IPFS,
+    IRequestParameters
+} from "../../net/src" // TODO reference the future package
 import { Buffer } from 'buffer/'
 import axios from "axios"
-import { Wallet, Signer } from "ethers"
-import { promiseWithTimeout } from "../util/timeout"
+import { Signer, Wallet } from "ethers"
+import { promiseWithTimeout } from "../../common/src" // TODO reference the future package
```

Comment:
This looks more like an utility than a common function

Reply:
You are right, but it's actually used in `net` and `client` package, therefore is in common. But totally agree that this should be considered as a util function. But I also have some doubts concerning this point that I want to discuss in a conversation :)